Revise setup and installation guide (#105)
recognai/rubrix: docs/getting_started/setup&installation.rst

In this guide, we will help you to get up and running with Rubrix. Basically, you need to:

1. Install the Python client

> 2. Launch the webapp

Reviewer: ```suggestion
2. Launch the web app
```

Separated sounds better to me

> 1. Install the Rubrix python client

Reviewer: ```suggestion
1. Install the Rubrix Python client
```

------------------------------------

First, make sure you have Python 3.6 or above installed.

Then you can install Rubrix with ``pip``\ :

.. code-block:: bash

   pip install rubrix

2. Setup and launch the webapp
------------------------------

There are two ways to launch the webapp:

#. Using `docker-compose <https://docs.docker.com/compose/>`_ (**recommended**).
#. Executing the server code manually

Using ``docker-compose`` (recommended)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

For this method you first need to install `Docker Compose <https://docs.docker.com/compose/install/>`_.
Then you can launch the docker-contained webapp with the following command:

.. code-block:: bash

   wget -O docker-compose.yml https://raw.githubusercontent.com/recognai/rubrix/master/docker-compose.yaml && docker-compose up

This is the recommended way because it automatically includes an
`Elasticsearch <https://www.elastic.co/elasticsearch/>`_ instance, Rubrix's main persistent layer.

Executing the server code manually
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> When executing the server coder manually you need to provide an

Reviewer: ```suggestion
When executing the server code manually you need to provide an
```

Isn't it code?

`Elasticsearch <https://www.elastic.co/elasticsearch/>`_ instance yourself.
This method may be preferred if you (1) want to avoid or cannot use ``Docker``,
(2) have an existing Elasticsearch service, or
(3) want to have full control over your Elasticsearch configuration.

1. First you need to install
   `Elasticsearch <https://www.elastic.co/guide/en/elasticsearch/reference/7.12/install-elasticsearch.html>`_
   and launch an Elasticsearch instance.
2. Install the Rubrix Python library together with its server dependencies:

.. code-block:: bash

   pip install rubrix[server]

> 3. Launch a local instance of the Rubrix webapp

Reviewer: ```suggestion
3. Launch a local instance of the Rubrix web app
```

Same as above

In this guide, we will help you to get up and running with Rubrix. Basically, you need to:

1. Install the Python client
2. Launch the web app

1. Install the Rubrix Python client
------------------------------------

First, make sure you have Python 3.6 or above installed.

Then you can install Rubrix with ``pip``\ :

.. code-block:: bash

   pip install rubrix

2. Setup and launch the webapp
------------------------------

There are two ways to launch the webapp:

#. Using `docker-compose <https://docs.docker.com/compose/>`_ (**recommended**).
#. Executing the server code manually

Using ``docker-compose`` (recommended)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

For this method you first need to install `Docker Compose <https://docs.docker.com/compose/install/>`_.
Then you can launch the docker-contained webapp with the following command:

.. code-block:: bash

>    wget -O docker-compose.yml https://raw.githubusercontent.com/recognai/rubrix/master/docker-compose.yaml && docker-compose up

Reviewer 2: The docker-compose installation should be a bit more detailed. Users should create a folder and download compose file there, for eventuals problems or app down actions. 

More like elasticsear open distro does: https://opendistro.github.io/for-elasticsearch/downloads.html#try
